chart/prometheus-msteams/Chart.yaml

```diff
@@ -3,7 +3,7 @@ appVersion: "v1.4.1"
 description: A Helm chart for Kubernetes
 name: prometheus-msteams
 home: https://github.com/prometheus-msteams/prometheus-msteams
-version: 0.6.0
+version: 0.6.1
```

Comment:
you don't need to bump this again. Please use `0.6.0`.

Reply:
Hang on...please don't change this version at all. I will create a new release and bump it, as `appVersion` will also change.